server/webhook_notifier.go

```diff
@@ -109,6 +110,24 @@ func (w *WebhookNotifier) NotifyConnected(ctx context.Context, clientAddr net.Ad
 	return w.send(ctx, payload)
 }
 
+func (w *WebhookNotifier) NotifyDisconnected(ctx context.Context, clientAddr net.Addr, serverAddress string, playerInfo *PlayerInfo, backendHostPort string) error {
+	if w.requireUser && playerInfo == nil {
+		return nil
+	}
+
+	payload := &WebhookNotifierPayload{
+		Event:           WebhookEventConnecting,
+		Timestamp:       time.Now(),
+		Status:          WebhookStatusDisconnected,
```

Comment:
...use WebhookStatusSuccess here. Maybe we could use a non-success status if the connection was dropped prematurely...however the code probably can't detect that difference right now.